openfisca_core/scenarios.py

```diff
@@ -15,6 +15,10 @@ def N_(message):
     return message
 
 
+def compare_periods(a, b):
+    return periods.compare_period_size(a, b) or periods.compare_period_start(a, b)
```

Comment:
Is there any value to sort the periods by start ? If no, we can just use `periods.compare_period_start` as `cmp` and avoid defining a local function.